Neos.Flow/Documentation/TheDefinitiveGuide/PartIII/AspectOrientedProgramming.rst

```diff
@@ -1033,23 +1034,30 @@ The following example introduces a new property "subtitle" to the class
 
 *Example: Property introduction*::
 
-	namespace Example\MyPackage;
+    namespace Example\MyPackage;
 
-	/**
-	 * An aspect for demonstrating property introductions
-	 *
-	 * @Flow\Aspect
-	 */
-	class PropertyIntroductionAspect {
+    use Neos\Flow\Annotations as Flow;
 
-		/**
-		 * @var string
-		 * @Column(length=40)
```

Comment:
Guess that never really worked anyway

Reply:
True. But even with `@ORM\Column` it won't work any longer…